pkg/networkservice/chains/nsmgr/scale_test.go

```diff
@@ -38,7 +38,7 @@ import (
 func TestCreateEndpointDuringRequest(t *testing.T) {
 	t.Cleanup(func() { goleak.VerifyNone(t) })
 
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5*10)
```

Comment:
Why are we suddenly 10x-ing all of the timeouts? What are we doing that is bloating the latency by 10x?

Reply:
It seems, that enabling full logs in tests running with race flag, increases time for tests. That's not something i'm going to actually push to sdk code, some changes only have been made for testing, so i'm converted this PR to draft, until it's ready.